config/software/patch.rb

```diff
@@ -15,22 +15,25 @@
 #
 
 name "patch"
-default_version "2.7"
 
-version("2.7") { source md5: "1cbaa223ff4991be9fae8ec1d11fb5ab" }
-
-source url: "http://ftp.gnu.org/gnu/patch/patch-#{version}.tar.gz"
-
-relative_path "patch-#{version}"
+if windows?
+  default_version "2.6.1"
+  dependency "mingw-get"
+else
+  default_version "2.7"
+  version("2.7") { source md5: "1cbaa223ff4991be9fae8ec1d11fb5ab" }
+  source url: "http://ftp.gnu.org/gnu/patch/patch-#{version}.tar.gz"
+  relative_path "patch-#{version}"
+end
 
 env = with_standard_compiler_flags(with_embedded_path)
 
 build do
-  configure_command = ["./configure",
-                       "--prefix=#{install_dir}/embedded",
-                       "--disable-xattr"]
-
-  command configure_command.join(" "), env: env
-  make "-j #{workers}", env: env
-  make "-j #{workers} install", env: env
+  if windows?
+    command "mingw-get.exe -v install msys-patch-bin=#{version}-*", env:env
```

Comment:
I'm worried about relying on what is listed as a GUI tool (`mingw-get`) to download various deps. Shouldn't we just fetch the source for these things directly and compile them?